relay-server/src/actors/project.rs

```diff
@@ -338,6 +344,20 @@ impl ProjectState {
     pub fn has_feature(&self, feature: Feature) -> bool {
         self.config.features.0.contains(&feature)
     }
+
+    pub fn measurements(&self) -> Option<&MeasurementsConfig> {
+        self.config
+            .measurements
+            .as_ref()
+            .or(self.global_config.measurements.as_ref())
+    }
+
+    pub fn metric_conditional_tagging(&self) -> impl Iterator<Item = &TaggingRule> {
```

Comment:
please double check if this is the order we want, I vaguely remember that global rules should come first